docs/fields/rich-text.mdx

```diff
@@ -77,6 +77,10 @@ The default `leaves` available in Payload are:
 
 Set this property to `true` to hide this field's gutter within the admin panel. The field gutter is rendered as a vertical line and padding, but often if this field is nested within a Group, Block, or Array, you may want to hide the gutter.
 
+**`upload.collections[collection-name].fields`**
+
+This allows [fields](/docs/fields/overview) to be saved as meta data on an upload field inside the Rich Text Editor. When this is present, the fields will render inside a modal that can be opened by clicking the "edit" button on the upload element.
+
```

Comment:
It would be great if we could add examples and show a screenshot of what this looks like